docs/api/asset.rst

```diff
@@ -1,7 +1,12 @@
 pystac.asset
 ============
 
-.. automodule:: pystac.asset
+.. autoclass:: pystac.asset.Asset
     :members:
     :undoc-members:
     :noindex:
+
+.. automodule:: pystac.asset
+    :members:
+    :undoc-members:
+    :exclude-members: Asset
```

Comment:
This pattern makes it so you can have the Asset indexed as `pystac.Asset` and the rest of the module as `pystac.asset.whatever`